slides/src/content/angular-cli.md

```diff
@@ -0,0 +1,56 @@
+# Using the Angular CLI
+
+---
+
+## What is Angular-cli
```

Comment:
What do you think about using lowercase for `angular-cli` and `ember-cli` to reflect the name of the npm package?